specs/commandline/sign.md

```diff
@@ -37,8 +37,44 @@ Flags:
       --plugin-config strings    {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
       --signature-format string  signature envelope format, options: 'jws', 'cose' (default "jws")
   -u, --username string          username for registry operations (default to $NOTATION_USERNAME if not specified)
+      --use-image-manifest       optional for using OCI image manifest to store signatures in a registry
 ```
 
+## Use OCI image manifest to store signatures
+
+By default, Notation uses [OCI artifact manifest][oci-image-manifest] to store signatures in the registries. For backward compatibility, Notation supports using `OCI image manifest` to store the signature in the registries that implement partial or older versions of the OCI Image specification. Use flag `--use-image-manifest` to force Notation store the signatures using OCI image manifest. Note that Notation doesn't automatically switch to `OCI image manifest` for a failure of putting `OCI artifact manifest`.
+
+Registries MAY not implement or enable the `Referrers API`, which is used by the client to fetch the referrers. In the context of Notation, the referrers are signatures. Notation follows the fallback procedure defined in [OCI distribution spec][oci-backward-compatibility] if `Referrers API` is unavailable.
+
+### Set config property for OCI image manifest
+
+OCI image manifest requires additional property `config` of type `descriptor`, which is not required by OCI artifact manifest. Notation creates a default config descriptor for the user if the flag `--use-image-manifest` is used.
+
+Notation uses empty JSON object `{}` as the default configuration content, and thus the default `config` property is fixed, as following:
```

Comment:
@nima Does ECR accept this config?

Reply:
Yes, I have a sample output which verifies this; keeping in mind for the following that `sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a` is the digest of `{}`:

```md
### (W) `PUT blobs/uploads/{orasConfig.blob}` :: `201 Created` ✅
#### Request
DEBU[0002]  Request URL: "https://080565210187.dkr.ecr.us-east-1.amazonaws.com/v2/utnubu/blobs/uploads/4fe80440-6088-4443-b805-c36a5c2ee156?digest=sha256%3A44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
DEBU[0002]  Request method: "PUT"
DEBU[0002]  Request headers:
DEBU[0002]    "Content-Type": "application/octet-stream"
DEBU[0002]    "Authorization": "*****"
DEBU[0002]    "User-Agent": "oras/0.16.0+Homebrew"
#### Response
DEBU[0002]  Response Status: "201 Created"
DEBU[0002]  Response headers:
DEBU[0002]    "Sizes": "1.000000"
DEBU[0002]    "Date": "Sun, 18 Dec 2022 01:32:30 GMT"
DEBU[0002]    "Content-Length": "0"
DEBU[0002]    "Docker-Content-Digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
DEBU[0002]    "Docker-Distribution-Api-Version": "registry/2.0"
DEBU[0002]    "Location": "https://080565210187.dkr.ecr.us-east-1.amazonaws.com/v2/utnubu/blobs/sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
```